pages/services/dcos-monitoring/1.0.1/operations/grafana/storage/index.md

```diff
@@ -12,13 +12,13 @@ model: ../../../data.yml
 
 ## DSS Volume
 
-You can configure Grafana to use a dedicated volume provided by the [DC/OS Storage Service](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/) (DSS).
+You can configure Grafana to use a dedicated volume provided by the [DC/OS Storage Service](https://docs.mesosphere.com/services/storage/1.0.0/) (DSS).
 
 At least one volume must be available for Grafana to use.
-For more information on creating volumes, see the [DC/OS Storage Service](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/cli-references/dcos-storage-volume/dcos-storage-volume-create) documentation.
+For more information on creating volumes, see the [DC/OS Storage Service](https://docs.mesosphere.com/services/storage/1.0.0/cli-references/dcos-storage-volume/dcos-storage-volume-create) documentation.
```

Comment:
it looks like there are also now 1.0.1 docs? I noticed the link version at 1.0.0 here

Reply:
where do you see 1.0.1? AFAIK we haven't tagged 1.0.0 final yet, so 1.0.1 should not yet exist.

Reply:
also, we should probably reach out to the dcos-monitoring team and update the docs in their repo since the docs here flow down stream from it

Reply:
@philipnrmn @gracedo @branden opend https://github.com/mesosphere/dcos-monitoring/pull/181 to pull the respective changes into the dcos-monitoring repo